rockspec/lua-resty-hmac-ffi-1.0-0.rockspec

```diff
@@ -0,0 +1,24 @@
+package = 'lua-resty-hmac-ffi'
+version = '1.0-0'
+
+source = {
+  url = 'git://github.com/jamesmarlowe/lua-resty-hmac',
+  tag = 'v1.0'
```

Comment:
Same question as above: any reason to choose a different value from the internal version number?

Reply:
replied you

Reply:
this value depends on the tag name.

eg: if you create a tag with version `0.05`, then I will update it to `0.05`.